types.go

```diff
@@ -55,6 +55,8 @@ type Config struct {
 	BuildArgs     []string `json:"build_args"      envconfig:"optional"`
 	BuildArgsFile string   `json:"build_args_file" envconfig:"optional"`
 
+	Mirrors []string `json:"mirrors" envconfig:"MIRRORS,optional"`
```

Comment:
Let me know if you prefer some other name for specifying mirrors.  I think I've also seen `registry_mirrors` elsewhere.